lib/Widget/RssProvider.php

```diff
@@ -60,14 +60,20 @@ public function fetchData(DataProviderInterface $dataProvider): WidgetProviderIn
             ->format('U');
 
         try {
+            $httpOptions = [
+                'headers' => [
+                    'Accept' => 'application/rss+xml, application/rdf+xml;q=0.8, application/atom+xml;q=0.6,'
+                        . 'application/xml;q=0.4, text/xml;q=0.4, text/html;q=0.2, text/*;q=0.1'
+                ],
+                'timeout' => 20, // wait no more than 20 seconds
+            ];
+
+            if (!empty($dataProvider->getProperty('userAgent'))) {
+                $httpOptions['User-Agent'] = $dataProvider->getProperty('userAgent');
```

Comment:
I might be wrong, please do double check with Guzzle documentation, but I *think* that the User-Agent option should be in headers
ie 
```
$httpOptions['headers']['User-Agent']
```

Reply:
You are right, it should be part of headers, Thanks for noticing that.